src/DataTypes/utils.ts

```diff
@@ -42,12 +43,16 @@ export const getDataTypeSchema = (
 			const: operatorKey,
 		}),
 	);
+	// Let's keep all ids generation here to have a better view.
+	schemaField.$id = `field-${index}`;
+	valueSchema.$id = `value-${index}`;
 	return {
+		$id: (schemaField?.title ?? '') + index,
```

Comment:
Should we add something as a fallback text for the id here?

Reply:
sure 🤦 schema title is not even appropriate